Makefile

```diff
@@ -92,11 +92,6 @@ GGML_METAL := 1
 DEPRECATE_WARNING := 1
 endif
 
-ifdef LLAMA_OPENMP
-GGML_OPENMP := 1
-DEPRECATE_WARNING := 1
-endif
-
```

Comment:
This should not be removed

Reply:
I think it should be removed, `GGML_OPENMP` is not an option in the `Makefile`. Since OpenMP is enabled by default, the only option the user can specify is `GGML_NO_OPENMP` to disable it.